docs/site-verification.md

```diff
@@ -0,0 +1,57 @@
+Site Verification
+=================
+
+After installing and [registering](common/registration.md) services from the [site planning document](site-planning.md),
+you will need to perform some verification steps before your site can
+[scale up to full production](#scale-up-to-full-production).
+
+Verify OSG Software
+-------------------
+
+To verify your site's installation of OSG Software, submit local test jobs, contact the OSG for end-to-end
+tests of pilot job submission, and check that OSG usage is reported to the [GRACC](https://gracc.opensciencegrid.org).
+
+### Local verification ###
+
+It is useful to submit jobs from within your site to verify CE's ability to submit jobs to your local batch system.
+Consult the document for [submitting jobs into an HTCondor-CE](compute-element/submit-htcondor-ce) for detailed
+instructions on how to test job submission.
+
+### Verify end-to-end pilot job submission ####
+
+Once you have validated job submission from within your site, request test pilot jobs from
+[OSG Operations](mailto:osg-gfactory-support@physics.ucsd.edu) and provide the following information:
+
+-   The fully qualified domain name of the CE
+-   Registered OSG resource name
+-   Supported OS version of your worker nodes (e.g., EL6, EL7, EL8, or a combination)
+-   Support for multicore jobs
+-   Support for GPUs
+-   Maximum job walltime
+-   Maximum job memory usage
+
+Once the Operations team has enough information, they will start submitting pilots to your CE.
+Initially, this will be a handful of pilots at a time but once the factory verifies that pilot jobs are running
+successfully, that number will be ramped up.
+
+### Verify reporting and monitoring ###
+
+To verify that your site is correctly reporting to the OSG, visit
+[OSG's Accounting Portal](https://gracc.opensciencegrid.org/dashboard/db/site-summary) and select your registered OSG
+site name from the `Site` dropdown.
+If you don't see your site in the dropdown,
+[please contact us for assistance](common/help.md#software-or-service-support).
+
```

Comment:
They don't need to check flock.opensciencegrid.org any more?

Reply:
Good catch but I did exclude that on purpose! Finding your site in the collection of images really isn't user friendly so I dropped it.